csharp/ql/src/semmle/code/csharp/Common.qll

```diff
@@ -0,0 +1,84 @@
+/**
+ * Provides classes and predicates for common functionality.
+ */
+import csharp
+
+/**
+ * Constructs a string containing the text `text` with a
+ * link to the element `e`.
+ */
+bindingset[text]
+string link(Element e, string text) {
```

Comment:
This isn't currently used or covered by a qltest.

Reply:
Sorry. We don't have a test for it ourselves. Do you want me to create one (it will take me some time as I am not the original author)